### DISCUSSION TITLE
adding alternative field names 'alias'

### DISCUSSION THREAD

**Discussion by voice1** [collerek/ormar#293] [Q&A] [answered]
Forgive me if this exists but if it does, I have not been able to locate any documentation for ormar.

When connecting to an existing database, that you can not alter the structure of, you sometimes need to use reserved keywords in python. such as a database with the field name of 'from'. 

In other ORM's there is the ability to specify a column parameter when creating the model so that you could use something like

```
class Table(ormar.Model):
        class Meta:
                pass

       from_: str = ormar.String(max_length=20, alias='from')
```

> I did try passing `alias=` and `field_name=` with no success.

SQLAlchemy supports something called [Mapping Columns](https://docs.sqlalchemy.org/en/14/orm/mapping_columns.html)
that solves this, but I do not see how to implement this with `ormar`.

Perhaps it is possible to add an attribute `column_name=` or 'alias=` to the column types?

**Comment by collerek** [comment, marked as answer]
The attribute you are looking for is simply `name` :)
That sets the db column name.

**Reply by voice1** [reply]
Thank you, Looking into the code it appears that `db_alias=` does the same thing, although `name=` is much more clearer. 

Might I suggest that the [common parameters](https://collerek.github.io/ormar/fields/common-parameters/) on the docs to include the `name=` / `db_alias=` options for future readers.

**Reply by collerek** [reply]
`db_alias` is how the value is stored internally (cause `name` is used for an `ormar` field name, and `alias` is already taken by pydantic).
So you should not use `db_alias` (it won't be saved from the constructor) but just `name`.

**Reply by collerek** [reply]
And yes, the name definitely should be included in common parameters, if you can issue a PR for this I would be grateful. If you don't have time I will add this later. Good point, thanks 😊
